tool/teleport/common/teleport.go

```diff
@@ -958,6 +958,9 @@ func onJoinOpenSSH(clf config.CommandLineFlags, conf *servicecfg.Config) error {
 func onIntegrationConfDeployService(params config.IntegrationConfDeployServiceIAM) error {
 	ctx := context.Background()
 
+	// Ensure we print output to the user. LogLevel at this point was set to Error.
+	log.SetLevel(log.InfoLevel)
```

Comment:
can't we reuse the same logger as teleport and change only if debug is set?
Not sure if users want to see info related logs when running something on cloud shell

Reply:
Yeah, I thought `-d` was global, but we actually don't have `-d` for integration commands. But I changed setting of the level to be flexible, so if the future there will be `-d` it will not get overwritten.

And I think showing those logs by default is helpful for the user, it tells useful information (policies names).